nilearn/regions/tests/test_hierarchical_kmeans_clustering.py

```diff
@@ -112,9 +112,13 @@ def test_hierarchical_k_means():
 
 
 def test_hierarchical_k_means_clustering():
-    data_img, mask_img = generate_fake_fmri(shape=(10, 11, 12), length=15)
+    n_samples = 15
+    n_features = 150
```

Comment:
I don't understand why is `n_features` 150 here?

Reply:
yup me neither but this is the n_features of the data we get... 

# 🤷🏽 ❓

Reply:
So that's after the inverse_transform, right? I think the shape is all wonky because of the `.T` you just removed

Reply:
not sure before the change I would get (15, 150) and after I just got (150,15)

or the other way around...

Reply:
ok so 150 is the n_features after masking